packages/schemas/tables/verification_statuses.sql

```diff
@@ -0,0 +1,16 @@
+create table verification_statuses (
```

Comment:
nit: Do we need a field to specify the verification method, like passcode or password?

Reply:
I don't really think we do, at least for now. Will see if we need to add it when we have a valid use case.